activerecord/lib/active_record/associations/nullified_owner_attributes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module ActiveRecord::Associations
+  module NullifiedOwnerAttributes
+    def nullified_owner_attributes
```

Comment:
Just enough to place it in `ForeignAssociation` since this method is needed for all has_* associations.